config/_production.js

```diff
@@ -1,6 +1,11 @@
-/* eslint key-spacing:0 */
-export default () => ({
-  compiler_fail_on_warning : false,
-  compiler_hash_type       : 'chunkhash',
-  compiler_devtool         : false
+import {setBackendUrl, setNoMock} from './_development'
+
+export default config => ({
+  compiler_hash_type: 'chunkhash',
+  compiler_devtool: false,
+  globals: {
+    ...config.globals,
+    __BACKEND_URL__: setBackendUrl(),
```

Comment:
isnt it broken now? if i dont set a backend url i expect it to be an empty string. does that work?